ocaml/xapi/records.ml

```diff
@@ -448,6 +448,15 @@ let net_record rpc session_id net =
         ~get:(fun () -> Record_util.network_default_locking_mode_to_string (x ()).API.network_default_locking_mode)
         ~set:(fun value -> Client.Network.set_default_locking_mode rpc session_id net
                  (Record_util.string_to_network_default_locking_mode value)) ();
+
+      make_field ~name:"purposes"
```

Comment:
@thomasmck just spotted that this should be "purpose", to make it consistent with the field name.

Reply:
@thomasmck  @thomassa  Gonna fix this now if no-one is working on it atm.

Reply:
@gaborigloi I'm not working on it

Reply:
Raised https://github.com/xapi-project/xen-api/pull/3261